src/components/PlaidLink/index.native.js

```diff
@@ -15,18 +18,17 @@ const PlaidLink = (props) => {
     useEffect(() => {
         openLink({
             tokenConfig: {
-                token: props.token,
+                token: propsRef.current.token,
             },
             onSuccess: ({publicToken, metadata}) => {
-                props.onSuccess({publicToken, metadata});
+                propsRef.current.onSuccess({publicToken, metadata});
             },
             onExit: (exitError, metadata) => {
                 Log.info('[PlaidLink] Exit: ', false, {exitError, metadata});
-                props.onExit();
+                propsRef.current.onExit();
             },
         });
     }, []);
```

Comment:
We don't need to use `useRef` here.
useEffect(() => {}, `[]`) - empty array dependency - this is equivalent to `componentDidMount`.
So `openLink` is called only once regardless of `props.token` change.

Reply:
interesting. but does it cause any harm? if we use the `props.token` directly then the `exhaustive-deps` linter would complain about a missing dependency.

Reply:
if it complains, we can disable with a comment instead of using ref: https://expensify.slack.com/archives/C01GTK53T8Q/p1680256754764159